### ISSUE TITLE
Exception reported: Language phrase "Open" not found (arg 3)

### ISSUE THREAD

**Issue by borzaka** [RoyZ-CSGO/Retakes_Breaker#4]
Latest version, compiled from master 2022.01.02.
I have installed the translation folder contents.
First starting up the server after installing the plugin:

```
[Retakes] All destructible entities have been destroyed.
L 01/02/2022 - 17:24:53: [SM] Exception reported: Language phrase "Open" not found (arg 3)
L 01/02/2022 - 17:24:53: [SM] Blaming: retakes.smx
L 01/02/2022 - 17:24:53: [SM] Call stack trace:
L 01/02/2022 - 17:24:53: [SM]   [0] FormatNativeString
L 01/02/2022 - 17:24:53: [SM]   [1] Line 67, ././scripting/retakes/natives.sp::Native_RetakeMessageToAll
L 01/02/2022 - 17:24:53: [SM]   [3] Retakes_MessageToAll
L 01/02/2022 - 17:24:53: [SM]   [4] Line 58, retakes_breaker.sp::Event_OnRoundStart
```

https://github.com/RoyZ-CSGO/Retakes_Breaker/blob/master/scripting/retakes_breaker.sp#L58

**Comment by enghausen** [comment]
@borzaka 
The https://github.com/RoyZ-CSGO/Retakes_Breaker/blob/master/translations/da/retakes_breaker.phrases.txt got syntax mistake. You have to remove the letter s outside the ""

```
"Phrases"
{
	"Breakable"
	{
		"da"			"Alle destruebare enheder er blevet ødelagt."
    }
    "Open"
    {
        "da"            "Alle døre er blevet åbnet."s
    }
}
```

Cheers!

**Comment by enghausen** [comment]
I am just tired today - just saw the https://github.com/RoyZ-CSGO/Retakes_Breaker/pull/3 now! Sorry!

**Comment by borzaka** [comment]
Fixed in #3
